go/engine/resolve_identify2.go

```diff
@@ -49,6 +49,19 @@ func (e *ResolveThenIdentify2) resolveUID(ctx *Context) (err error) {
 		return nil
 	}
 
+	// if no uid and no assertion, then if logged in use self uid:
+	if len(e.arg.UserAssertion) == 0 {
```

Comment:
are we sure we always want this behavior?  It makes sense in the context of `keybase id` but I'm not sure if it does everywhere else?